IntegrationAdaptersTests/tests/test.txt

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace IntegrationAdaptersTests.tests

```

Comment:
what is this file? 😄 it's .txt

Reply:
This is generated only for tests. I forgot to change the contents of the file. We will correct it.